src/Metrics.Extensions/src/Public/Microsoft.ApplicationInsights.Metrics/MetricSeriesConfigurationForAccumulator.cs

```diff
@@ -0,0 +1,200 @@
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public sealed class AggregateKindConstants
+        {
+            internal static readonly AggregateKindConstants Instance = new AggregateKindConstants();
+
+            private AggregateKindConstants()
+            {
+            }
+
+            /// <summary>
+            /// The kind moniker for aggregates produced by aggregators that are configured by metric configurations represented
+            /// through instances of <see cref="MetricSeriesConfigurationForAccumulator"/>.
+            /// </summary>
+            public string AggregateKindMoniker { get { return Constants.AggregateKindMoniker; } }
+
+            /// <summary>
+            /// Constants used to refer to data fields contained within aggregates produced by aggregators that are configured
+            /// by metric configurations represented through instances of <see cref="MetricSeriesConfigurationForAccumulator"/>.
+            /// </summary>
+            public DataKeysConstants AggregateKindDataKeys { get { return DataKeysConstants.Instance; } }
+
+            /// <summary>
+            /// Groups constants used to refer to data fields contained within aggregates produced by aggregators that are configured
+            /// by metric configurations represented through instances of <see cref="MetricSeriesConfigurationForAccumulator"/>.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public sealed class DataKeysConstants
+            {
+                internal static readonly DataKeysConstants Instance = new DataKeysConstants();
+
+                private DataKeysConstants()
+                {
+                }
+
+                /// <summary>
+                /// The name of the Sum field in <see cref="MetricAggregate"/> objects produced by accumulator aggregators.
+                /// </summary>
+                public string Sum { get { return Constants.AggregateKindDataKeys.Sum; } }
+
+                /// <summary>
+                /// The name of the Minimum field in <see cref="MetricAggregate"/> objects produced by accumulator aggregators.
+                /// </summary>
+                public string Min { get { return Constants.AggregateKindDataKeys.Min; } }
+
+                /// <summary>
+                /// The name of the Maximum field in <see cref="MetricAggregate"/> objects produced by accumulator aggregators.
+                /// </summary>
+                public string Max { get { return Constants.AggregateKindDataKeys.Max; } }
+            }
+        }
+
+        /// <summary>
+        /// Defines constants used my metric aggregates produced by aggregators that are configured by metric configurations represented
+        /// through instances of <see cref="MetricSeriesConfigurationForAccumulator"/>.
+        /// </summary>
+        public static class Constants
```

Comment:
Why this is browsable than?

Reply:
There are examples in
https://github.com/Microsoft/ApplicationInsights-dotnet/blob/feature/PreAggMetrics.Master/src/MetricsTests/Microsoft.ApplicationInsights.Metrics.Core.Test/Examples.cs